### ISSUE TITLE
Option to fork lab is missing

### ISSUE THREAD

**Issue by NColey** [learn-co-curriculum/manipulating-hashes#16]
The option to fork this lab, check when build tests are passing, or see when the pull request has gone through successfully are missing. They seem to be missing in the learn app as well (the options are greyed out) but I was able to access them by selecting 'view on Github' so that I could complete this section before moving to the next section. I believe the right hand side of the lab should look like this:

![expected](https://cloud.githubusercontent.com/assets/8762012/9834305/cca3ffd6-5982-11e5-91f6-e233d6dc17f2.png)

but instead the lab looks like this:

![received](https://cloud.githubusercontent.com/assets/8762012/9834307/d8ac1dea-5982-11e5-8185-229fea4366c1.png)

This allows you to move on without completing the lab if you don't think to find it through Github (and the way the options are greyed out in the Learn app makes it look like there isn't anything that can be done to complete the lab). If this lab could be updated to include those options that would be great! Thanks.

**Comment by NColey** [comment]
Just an update, after I completed this lab I moved on to the next one (Manipulation Lab) and in the terminal I got the following error: "WARNING: You are attempting to open a lesson that is beyond your current lesson. Are you sure you want to continue? [Yn]: Y" . I think this is related to the existing issue. While using learn in the terminal showed that I completed everything fine maybe not being able to wait for all the lights to turn green under 'local build' and 'pull request' (because they aren't there) marks the lab as incomplete. Just wanted to add this to the existing issue because I think they are related.
